tendrl/node_agent/collectd/collectors/ceph/tendrl_ceph_cluster_iops.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
```

Comment:
No the way my plugin framework works is with just 1 configuration for gluster or ceph, all respective plugins will be dynamically loaded and executed.so for that I need that bifurcation of plugins